tests/snuba/api/endpoints/test_organization_events_mep.py

```diff
@@ -3207,6 +3207,60 @@ def test_transaction_user_misery(self) -> None:
             },
         }
 
+    def _setup_user_misery(self, spec: OnDemandMetricSpec) -> None:
+        events = [
+            ("one", 300),
+            ("two", 300),
+            ("one", 3000),
+            ("two", 3000),
+            ("three", 400),
+            ("four", 4000),
+        ]
+        for index, event in enumerate(events):
+            email = f"{event[0]}@example.com"
+            tags = {"user.email": email}
+            if event[1] > 300 * 4:
+                tags[constants.METRIC_SATISFACTION_TAG_KEY] = constants.METRIC_FRUSTRATED_TAG_VALUE
+            self.store_on_demand_metric(
+                index,
```

Comment:
Careful, I think this is what is not adding up to 6. If it's a set then you are getting a sum value for this I think passed as the value. Try changing this to 1 perhaps.